spec/forms/hub/create_client_form_spec.rb

```diff
@@ -213,6 +213,62 @@
         end
       end
 
+      context "contact opt in -- no opted into methods" do
```

Comment:
since several forms (all children of `client_form`) changed as part of this PR, it would probably be good to test those forms in a similar way. it could be a bit redundant so maybe a shared example would work? this can be a follow-up task also